rdopkg_tar/tar_changes.py

```diff
@@ -201,12 +211,8 @@ def main():
 
     # Check for the original commiter username in jenkins env variables.
     # If it exists, then amend a suffix to the commit.
```

Comment:
Let's remove this comment, since it's now duplicated (copy-pasted) in the method above:
```suggestion
```